tools/ci-checks/test_demos_file_structure.py

```diff
@@ -77,7 +79,7 @@ def test_file_structure():
     for sub_folder in os.listdir('demos'):
         folder = os.path.join(root_folder, sub_folder)
         if os.path.isdir(folder):
-            logging.info('Testing folder: ' + folder)
+            logging.info('Testing folder: %s', folder)
```

Comment:
Is this pep8 or just about style (concat string vs. using format capabilities)

Reply:
This is a well know pitfall ... logger formatter will catch errors while formatting concatenation witll raise them ... see example discussion [here](https://reinout.vanrees.org/weblog/2015/06/05/logging-formatting.html)